src/winuwp/runner_uwp/CMakeLists.txt

```diff
@@ -0,0 +1,127 @@
+﻿cmake_minimum_required (VERSION 3.8)
+set(CMAKE_SYSTEM_NAME WindowsStore)
+set(CMAKE_SYSTEM_VERSION 10.0)
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED YES)
+
+include(CMakePrintHelpers)
+
+project (runner LANGUAGES CXX)
+
+# UWP tile and icon assets.
+set(ASSET_FILES ${ASSET_FILES}
```

Comment:
To avoid including this in the git repo history, we can upload them to the flutter_template_images package: https://pub.dev/packages/flutter_template_images